app/src/androidTest/java/org/mozilla/fenix/ui/robots/BrowserRobot.kt

```diff
@@ -570,13 +570,15 @@ class BrowserRobot {
         tabCrashRestoreButton.click()
     }
 
-    fun fillAndSubmitGoogleSearchQuery(searchString: String) {
+    fun fillAndSubmitDuckDuckGoSearchQuery(searchString: String) {
```

Comment:
could you provide some insight into why this change is needed, please? Maybe also on the github issue or PR description?

Reply:
Thanks for clarifying. That's the reason I asked, because I know that duckDuckGo also blocks suspicious traffic from my experience with adding custom search engines. I propose to use one of the testapp pages as external link (even if it doesn't cover the manual test steps 100%).

Reply:
Done!